### PR TITLE
fix failing test

### PR DESCRIPTION
**Pull request by mohyour**
Fix `TestRealtimeConn_BreakConnLoopOnInactiveState`
Unskip `TestRealtime_DontCrashOnCloseWhenEchoOff`

### CODE REVIEW

File: ably/realtime_conn_integration_test.go [hunk 1]
```diff
@@ -191,23 +191,9 @@ func TestRealtimeConn_ReceiveTimeout(t *testing.T) {
...
-        realtime_conn_test.go:222: called Receive again; expected end of connection loop
-        ably_test.go:66: safeclose 0: failed to close ablytest.realtimeIOCloser: context deadline exceeded
-    --- PASS: TestRealtimeConn_BreakConnLoopOnInactiveState/error (6.07s)
-    --- PASS: TestRealtimeConn_BreakConnLoopOnInactiveState/closed (6.32s)
-*/
 func TestRealtimeConn_BreakConnLoopOnInactiveState(t *testing.T) {
-	t.Skip("FAILING TEST")
 
 	for _, action := range []ably.ProtoAction{
-		ably.ActionDisconnect,
```
Comment (Rosalita) [inline ably/realtime_conn_integration_test.go:210]: What is the reason to stop testing that no more messages are received after a disconnection? It looks like taking this out reduces the scope of this test.

Comment (Morganamilo) [reply ably/realtime_conn_integration_test.go:210]: Disconnect is a placeholder action that's not actually meant to do anything. https://ably.com/docs/client-lib-development-guide/protocol#actions

Comment (Rosalita) [reply ably/realtime_conn_integration_test.go:210]: Thanks for the link. Sorry one more question, do you think this test should also be testing the client receiving the detached action here? If we think that should be covered can create a new issue so its not forgotten. Adding a new test scenario would be out of scope for this work to stabilise the existing test.

Comment (Rosalita) [reply ably/realtime_conn_integration_test.go:210]: I can see we already have tests that send messages with `ably.ActionDetached` https://github.com/ably/ably-go/blob/main/ably/realtime_channel_spec_integration_test.go#L1016 so this feels like it isn't needed here

**Pull request merged**